i18n/en-US.js

```diff
@@ -273,9 +273,9 @@ module.exports = {
     },
     title: '{nuxt} Blog',
     description:
-      'Discover articles from the {nuxtTeam} and {ambassadors} about NuxtJS, tips and tricks included!',
+      'Discover articles from the {nuxtTeam} and {nuxtCommunity} about NuxtJS, tips and tricks included!',
     nuxt_team: 'NuxtJS Team',
-    ambassadors: 'NuxtJS Community',
+    nuxt_community: 'NuxtJS Community',
```

Comment:
this needs to be Nuxt.js Community not NuxtJS. NuxtJS is for company and Nuxt.js is for open source. You will see it on the page https://nuxtjs.org/blog
Thanks 😊

Reply:
I just found out about it 😅
i will change it back